openreview/conference/invitation.py

```diff
@@ -350,6 +354,17 @@ def __init__(self, conference, note):
 
         withdraw_process_file = 'templates/withdraw_process.py'
 
+
+        if conference.submission_stage.reveal_submissions_on_withdraw:
+            readers = {
+                'description': 'User groups that will be able to read this withdraw note.',
+                'values': ['everyone']
+            }
+        else:
+            readers = {
+                'values-copied': [conference.id, '{signatures}']
```

Comment:
Maybe we can include the paper author group here, instead of just the signature of the withdraw comment note.

Reply:
you are right, because this is an invitation per paper, right?

Reply:
Correct. This invitation creates a reply note on the forum and changes the invitation of the forum to a conference level invitation for all withdrawn papers.
So, it would be apt to show this reply note to all the authors of this paper.

Reply:
The paper authors are still not able to see the desk reject note.

<img width="1152" alt="Screen Shot 2020-02-13 at 10 35 29 AM" src="https://user-images.githubusercontent.com/13500158/74450732-94412600-4e4c-11ea-8b97-009f4006cff3.png">

Reply:
ahhh good catch, I'll fix it.

Reply:
do you think we need to add all the members of the committee: assigned AC, assigned Reviewer

Reply:
I'm not certain about that. Ideally, we should list them as readers. 
But, Reviewers and ACs would not have a direct link to this paper on their consoles anymore, since all the consoles get assignments by querying for notes responding to `Submission` or `Blind_Submission` invitation.
So, I don't think adding them as readers would have an impact.

Reply:
I'm setting the readers of the desk reject note with the same value than the blinded note.

Reply:
same for withdraw

Reply:
Sounds good.